### PR TITLE
Clarify deck error when it fails to resolve GCS path.

### LINKED ISSUE #15491: Deck's view handler does not provide sufficient logging to debug
What is this trying to resolve? 
/area prow
/area prow/deck
/kind oncall-hotlist
/assign @Katharine @michelle192837 

```yaml
{
 insertId: "w6th29fa3buoi"  
 jsonPayload: {
  component: "deck"   
  error: "error when resolving real path: gcs path should have both a bucket and a path"   
  file: "prow/cmd/deck/main.go:930"   
  func: "main.handleRequestJobViews.func1"   
  handler: "/view"   
  level: "error"   
  msg: "error rendering spyglass page"   
 }
 labels: {
  compute.googleapis.com/resource_name: "gke-prow-default-pool-116f5d3e-csc4"   
  container.googleapis.com/namespace_name: "default"   
  container.googleapis.com/pod_name: "deck-7596cd9659-n6lv9"   
  container.googleapis.com/stream: "stderr"   
 }
 logName: "projects/k8s-prow/logs/deck"  
 receiveTimestamp: "2019-12-05T18:17:52.768123622Z"  
 resource: {
  labels: {
   cluster_name: "prow"    
   container_name: "deck"    
   instance_id: "6181871745597929897"    
   namespace_id: "default"    
   pod_id: "deck-7596cd9659-n6lv9"    
   project_id: "k8s-prow"    
   zone: "us-central1-f"    
  }
  type: "container"   
 }
 severity: "ERROR"  
 timestamp: "2019-12-05T18:17:50Z"  
}
```

### PR DESCRIPTION
**Pull request by michelle192837**
fixes #15491 

/assign @fejta @Katharine @cjwagner

### CODE REVIEW

File: prow/cmd/deck/main.go [hunk 1]
```diff
@@ -912,7 +912,7 @@ func renderSpyglass(sg *spyglass.Spyglass, cfg config.Getter, src string, o opti
 	src = strings.TrimSuffix(src, "/")
 	realPath, err := sg.ResolveSymlink(src)
 	if err != nil {
-		return "", fmt.Errorf("error when resolving real path: %v", err)
+		return "", fmt.Errorf("error when resolving real path '%v': %v", src, err)
```
Comment (fejta) [inline prow/cmd/deck/main.go]: Consider [wrapping](https://blog.golang.org/go1.13-errors#TOC_3.3.) errors in the future with `%w`

src is a string so I'd use `%s` or `%q`

**Comment by michelle192837** [comment]
Welp that's not what I meant to do. :/ Fixing.

**Comment by michelle192837** [comment]
Mucked this up and it's on master, whoops. Closing this and re-opening in https://github.com/kubernetes/test-infra/pull/15629.

File: prow/cmd/deck/main.go (hunk 1 above)
Comment (michelle192837) [reply prow/cmd/deck/main.go]: Ack'd

**Pull request closed without merging**